examples/xonotic/Makefile

```diff
@@ -23,12 +23,41 @@
 #     \_/ \__,_|_|  |_|\__,_|_.__/|_|\___|___/
 #
 
-REPOSITORY = us-docker.pkg.dev/agones-images/examples
+REPOSITORY ?= us-docker.pkg.dev/agones-images/examples
+
+WINDOWS_VERSIONS = ltsc2019 ltsc2022
+BUILDX_WINDOWS_BUILDER = windows-builder
+
+# Build with Windows support
+WITH_WINDOWS ?= 1
+# Build with ARM64 support
+WITH_ARM64 ?= 1
+
+# When pushing set to --push
+WINDOWS_DOCKER_PUSH_ARGS = --push
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
+server_tag = $(REPOSITORY)/xonotic-example:1.2
+
+ifeq ($(WITH_WINDOWS), 1)
+server_tag_linux_amd64 = $(server_tag)-linux-amd64
+server_tag_linux_arm64 = $(server_tag)-linux-arm64
+else
+server_tag_linux_amd64 = $(server_tag)-amd64
+server_tag_linux_arm64 = $(server_tag)-arm64
+endif
+
+push_server_manifest = $(server_tag_linux_amd64)
 root_path := $(realpath $(project_path)/../..)
-image_tag = $(REPOSITORY)/xonotic-example:1.1
+
+
+ifeq ($(WITH_WINDOWS), 1)
+push_server_manifest += $(foreach windows_version, $(WINDOWS_VERSIONS), $(server_tag)-windows_amd64-$(windows_version))
+endif
+ifeq ($(WITH_ARM64), 1)
```

Comment:
I just double checked: https://xonotic.org/download/

And Xonotic doesn't support Arm, only x86-64 - so let's remove the arm support please.

Reply:
> I just double checked: https://xonotic.org/download/
> 
> And Xonotic doesn't support Arm, only x86-64 - so let's remove the arm support please.

Sure, I will remove it.